### DISCUSSION TITLE
Bypass rendering the waveform

### DISCUSSION THREAD

**Discussion by nickgerig** [katspaugh/wavesurfer.js#3573] [General]
In certain situations I'd like to not render a waveform. For example, if I have a large file and no pre-generated peaks data, then I know that this will cause a OOM for the browser. In this case I'd like to retain all the hooks and UI logic I have attached to wavesurfer and just render a slider instead of a waveform. Is there an easy way to do that?

**Comment by katspaugh** [comment]
You could try passing peaks: [] in the options.

**Reply by nickgerig** [reply]
I did try that but there are some checks performed on the peaks array that cause an exception. I just tried passing in a fake array equal to the length of the file and that seems to work with also setting the wave height to 0.  

`const defaultPeaks = {
  data: new Array(1000).fill(0),
  length: 1000,
}`

**Reply by katspaugh** [reply]
You can also fill it with 0.01 if you want the waveform to be a thin line.
